apps/condo/domains/organization/schema/SendOrganizationEmployeeRequestService.test.js

```diff
@@ -0,0 +1,274 @@
+                mutation: 'sendOrganizationEmployeeRequest',
+                variable: ['data'],
+                code: 'BAD_USER_INPUT',
+                type: 'EMPLOYEE_INVITATION_ALREADY_SENT',
+                message: 'The invitation has already been sent to the employee.',
+            }, 'result')
+        })
+
+        test('should throw error if the request to one organization has already been sent 5 times', async () => {
+            for (let i = 0; i < 5; i++) {
+                const [request] = await sendOrganizationEmployeeRequestByTestClient(staff, {
+                    organization: { id: organization.id },
+                })
+                await updateTestOrganizationEmployeeRequest(admin, request.id, { isRejected: true })
+            }
+            await expectToThrowGQLError(async () => {
+                await sendOrganizationEmployeeRequestByTestClient(staff, {
+                    organization: { id: organization.id },
+                })
+            }, {
+                mutation: 'sendOrganizationEmployeeRequest',
+                variable: ['data'],
+                code: 'BAD_USER_INPUT',
+                type: 'REQUEST_TO_ORGANIZATION_LIMIT_REACHED',
+                message: 'A request to the organization limit reached',
+            }, 'result')
+        })
+
+        test('should throw error if the request has not yet been decided', async () => {
+            await sendOrganizationEmployeeRequestByTestClient(staff, {
+                organization: { id: organization.id },
+            })
+            await expectToThrowGQLError(async () => {
+                await sendOrganizationEmployeeRequestByTestClient(staff, {
+                    organization: { id: organization.id },
+                })
+            }, {
+                mutation: 'sendOrganizationEmployeeRequest',
+                variable: ['data'],
+                code: 'BAD_USER_INPUT',
+                type: 'REQUEST_NOT_DECIDED',
+                message: 'A request not decided yet. Please wait for a decide on the request from the organization',
+            }, 'result')
+        })
+
+        test('should throw error if authed user has not phone', async () => {
+            const staff = await makeClientWithStaffUser({ phone: null })
+
+            await expectToThrowGQLError(async () => {
+                await sendOrganizationEmployeeRequestByTestClient(staff, {
+                    organization: { id: organization.id },
+                })
+            }, {
+                mutation: 'sendOrganizationEmployeeRequest',
+                variable: ['data'],
+                code: 'BAD_USER_INPUT',
+                type: 'USER_DOES_NOT_HAVE_PHONE',
+                message: 'The user does not have a phone',
+            }, 'result')
+        })
+
+        test('should throw error if user send many requests (more then 100)', async () => {
+            const staff = await makeClientWithStaffUser()
+
+            for (let i = 0; i < 100; i++) {
```

Comment:
you need to move values like this (100) in const and reuse it for update it's in one place